.eslintrc.json

```diff
@@ -0,0 +1,57 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:import/errors",
+        "plugin:import/warnings"
+    ],
+    "overrides": [
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react",
+        "react-hooks",
+        "@typescript-eslint",
+        "import"
+    ],
+    "rules": {
+        "import/order": [
+            "error",
+            {
+                "newlines-between": "ignore",
+                "groups": [
+                    "builtin",
+                    "external",
+                    "internal",
+                    ["parent", "sibling", "index"],
+                    "object"
+                ],
```

Comment:
Below the groups add:

```js
pathGroups: [
  {
    pattern: 'react',
    group: 'external',
    position: 'before',
  },
],
pathGroupsExcludedImportTypes: ['react'],
```

That means the React imports will have priority over all **external** imports that are alphabetically sorted.

Reply:
I added it in [commit 8](https://github.com/zuko4celik/movies-app-new/pull/20/commits/111c24239002fb42784cca5cadad7697b599c6dd).